medcat/cdb.py

```diff
@@ -305,6 +312,7 @@ def add_addl_info(self, name: str, data: Dict, reset_existing: bool = False) ->
             self.addl_info[name] = {}
 
         self.addl_info[name].update(data)
+        self.is_dirty = True
```

Comment:
can you add these dirty flag flips to the other methods on the CDB. such as:
- import_training
- reset_cui_count
- reset_training
- filter_by_cui
- reset_cconcept_similarity